GsaGH/Parameters/5_Results/LineResultGoo.cs

```diff
@@ -13,49 +13,50 @@ namespace GsaGH.Parameters
   /// </summary>
   public class LineResultGoo : GH_GeometricGoo<Line>, IGH_PreviewData
   {
-    public LineResultGoo(Line line, IQuantity result1, IQuantity result2, Color color1, Color color2, float size1, float size2) : base(line)
+    public int ElementId;
+    public IQuantity Result1;
```

Comment:
do we want to change it outside the class?

Reply:
users might want to access this via code from grasshopper (c# / python)